source/release-notes/backend.txt

```diff
@@ -13,6 +13,30 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240605:
+
+5 June 2024 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Support for nested collections in ``Mixed`` properties is now available. To learn more, refer to :ref:`<collections-in-mixed>`. 
+
+- Improvements to the Function Editor for all Apps, including the following changes:
+
+  - A new :guilabel:`Format Code` button to format function code.
+
+  - A new :guilabel:`Clear & Run` button that replace the :guilabel:`Clear Result` button.
+  
+  - Autocomplete for common functionalities and App Services context.
+
+- App error logs now include GraphQL failures.
+
+- Users can no longer disable client max offline time. Users can continue to increase or decrease
+  the number of days for client max offline time where necessary.
+
+- Device Sync apps with an indexed queryable field now support schemas that do not contain the indexed queryable field. 
```

Comment:
sorry if a stupid question, but is this supposed to say something like this:
```suggestion
- Device Sync now supports schemas that do not contain the indexed queryable field. 
```
otherwise, I am pretty confused as to what this feature is actually doing 🙃

Reply:
Previously, when an app had an IQF, all schemas for the app had to have the IQF as a property. Will add a sentence about the previous behavior to explain.